app/views/shelters/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Shelters</h1>
+
+<% @shelters.each do |shelter| %>
+    <h2><%= shelter.name %></h2>
```

Comment:
Currently shelters are just header elements, future iterations will make these into clickable links that take us to the shelter's page.